milli/src/update/settings.rs

```diff
@@ -933,15 +943,19 @@ impl<'a, 't, 'i> Settings<'a, 't, 'i> {
                 {
                     match joined {
                         // updated config
-                        EitherOrBoth::Both((name, mut old), (_, new)) => {
+                        EitherOrBoth::Both((name, (mut old, user_defined)), (_, new)) => {
                             changed |= EmbeddingSettings::apply_and_need_reindex(&mut old, new);
                             if changed {
-                                tracing::debug!(embedder = name, "need reindex");
+                                tracing::debug!(
+                                    embedder = name,
+                                    documents = user_defined.len(),
```

Comment:
nitpick, but shouldn't `documents` equal the number of documents that are **not** user defined?

Reply:
Yes but we don't have the info, I changed `documents` to `user_provided` to make it clear 👍